src/test/xmeta/CMakeLists.txt

```diff
@@ -45,7 +45,13 @@ if (MSVC)
     target_link_libraries(test_xmeta windowsapp ole32 WindowsApp.lib comsuppw.lib)
 endif()
 
-target_sources(test_xmeta PUBLIC main.cpp)
+get_target_property(xmeta_path foundation_metadata Foundation_xmeta)
+add_custom_command(TARGET test_xmeta POST_BUILD
```

Comment:
Should we add be running these tests as part of our Azure pipelines build?

Reply:
I don't see why not. You will have to show me how to set it up though.